.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature Request
-about: Suggest an idea for Numba
+about: Tell us about something that it'd be good for Numba to support
```

Comment:
```suggestion
about: Tell us about something that would be good for Numba to support
```

Reply:
Actually the sentence now sounds even weirder, maybe we need a different approach entirely.

Reply:
How about "Tell us about something you'd like to see supported in Numba" ?

Reply:
That does sound more fluid, yes.

Reply:
Or even "Tell us about something in the Python language/NumPy you'd like to see supported in Numba" given that this is opening a feature request in the tracker, limiting the scope to simple "X is missing" type things might be a good idea as more involved stuff is already being redirected towards discourse?

Reply:
How about "Tell us about something in the Python language/NumPy you'd like Numba to support"?

Reply:
I like that! Thanks @gmarkall

Reply:
e496e19  has this